### ISSUE TITLE
Installation instructions out of date

### ISSUE THREAD

**Issue by GijsTimmers** [hbekel/bashrun2#3]
Can't install bashrun2 with `./configure` as specified on the website, `INSTALL` and `README`.

**Comment by hbekel** [comment]
Works fine for me. Since you don't give any more information other than "can't", there's nothing I can do about this.

**Comment by GijsTimmers** [comment]
This is what I'm getting:

```
$ ./configure
bash: ./configure: No such file or directory
```

**Comment by jm42** [comment]
@GijsTimmers Here are all the steps you need to run to install it.

```sh
$ curl -O 'http://henning-bekel.de/download/bashrun2/bashrun2-0.2.3.tar.gz'
$ tar xvf bashrun2-0.2.3.tar.gz
$ cd bashrun2-0.2.3/
$ ./configure --prefix=/usr --sysconfdir=/etc
$ make
$ sudo make install
```

**Comment by GijsTimmers** [comment]
@jm42 works! thanks!

**Comment by hbekel** [comment]
The repository now contains the configure script as well. Before moving this to github I used to distribute the result of make dist, which contains configure by default.

Running autoreconf would have done the trick as well.

**Comment by hbekel** [comment]
Fixed in f8048d2, release pending
